src/pages/Profile.js

```diff
@@ -1,8 +1,32 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchMissions } from '../redux/missions/missions';
+import './Profile.css';
 
 function Profile() {
+  const missions = useSelector((state) => (state.missions));
+
+  const dispatch = useDispatch();
+
```

Comment:
Only missions that should be displayed on the profile should be the joined mission, I believe here you call all the missions,

Reply:
Good catch :+1: